src/main/scala/nl/knaw/dans/easy/multiDeposit/DDM.scala

```diff
@@ -73,7 +73,25 @@ object DDM {
       "DCX_RELATION_TITLE" -> "",
       "DCX_RELATION_LINK" -> "")
 
-  val allFields = "ROW" :: "DATASET_ID" ::
+  // TODO not sure about some of the fields, see inline:
```

Comment:
@janvanmansum do the current fields suffice or do we also want the other 4 below? Couldn't find a reference to them yet in the DDM based on [dcterms](http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=terms#)